docs/glossary.rst

```diff
@@ -0,0 +1,152 @@
+==============
+Solar Glossary
+==============
+
+Resources
+==========
+
+Inputs
+--------
+Resource configuration that will be used in actions, handlers and orchestration.
+All known inputs should be provided in meta.yaml ::
+
+    input:
+      apache_ports:
+        value: null
+      fqdn:
+        value: null
+      puppet_modules:
+        value: null
+      role:
+        value: null
+
+Connections
+------------
+Allows to build hierarchy between inputs of several resources,
+parent value will be always used in child while connection is created.
+If connection will be removed - original value of child will be preserved.
+
+Actions
+--------
+Solar wraps deployment code into actions with specific names.
+Several actions of resource are mandatory:
+- run
+- remove
+- update
+
+All actions should be provided in meta.yaml ::
+
+    actions:
+      run: run.pp
+      update: run.pp
+
+Tags
+------
+Used to create arbitrary groups of resources, later this groups will be
+used for different user operations.
+
+Handlers
+=========
+Layer that responsible for action execution and tracking result.
+Currently handler specified in resource meta.yaml and used for all resource
+actions ::
+
+  handler: puppet
```

Comment:
maybe glossary shouldnt have a lot of details about each available handler?
i think right way would be to include link to handler speficic doc